spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    context 'when passing too long password' do
+      it 'validates password length in bytes' do
```

Comment:
you might want to consider using the aggregates failure flag
```suggestion
      it 'validates password length in bytes', :aggregate_failures do
```
This will return all failures in this block with reference to what went wrong otherwise it may be difficult to find the failure with mulltiple expects in a single it

Reply:
thanks for the hint, I will add it:)